src/commons/workspaces-menu/workspace-icon.js

```diff
@@ -0,0 +1,31 @@
+import {inject, customElement, bindable} from 'aurelia-framework';
+import {EventAggregator} from 'aurelia-event-aggregator';
+import {autoinject} from "aurelia-dependency-injection";
+
+@inject(EventAggregator)
+@inject(Element)
```

Comment:
same here regarding injects

Reply:
workspace icon still not ready yet, just random code there